packages/extension/src/store/modules/ui.js

```diff
@@ -18,6 +18,7 @@ const initialState = () => ({
   lastBannerSeen: new Date(),
   showPremium: false,
   showNewSource: false,
+  openNewTab: false,
```

Comment:
I think it's better to set it to `true` by default.
Many users asked why we changed this behavior.